### DISCUSSION TITLE
Can not run DPA-2 pretrained model for finetuning

### DISCUSSION THREAD

**Discussion by Zch102xjtumse** [deepmodeling/deepmd-kit#4124] [Q&A] [answered]
I want to use the pretrained model for finetuning basing my own metals datasets in 'https://www.aissquare.com/models/detail?pageType=models&name=DPA-2.2.0-v3.0.0b3&id=272' and the JSON file was obtained in 'https://bohrium.dp.tech/notebooks/2649266844' which I replaced the training sets and validation sets with my own dataset.But it doesn't work.The error is 'dargs.dargs.ArgumentValueError: [at location `model/descriptor/list/1`] get invalid choice `se_uni` for flag key `type`.Did you mean: se_t?'
Is there any wrong with my input JSON file?

**Comment by njzjz** [comment]
This JSON file is incompatible with DeePMD-kit.

@Chengqian-Zhang could you make clarification in your notebook?

**Reply by Chengqian-Zhang** [reply]
It's a version issue, I'll update the notebook version today

**Comment by Chengqian-Zhang** [comment, marked as answer]
Hi, you can read this notebook: https://bohrium.dp.tech/notebooks/18475433825. The version of deepmd-kit for this notebook is `beta3`.

**Reply by Zch102xjtumse** [reply]
Hi，I'm grateful for your assistance,but when using the input.json for singletask, I met this problem: dargs.dargs.ArgumentKeyError: [at location `model/descriptor`] undefined key `tebd_dim` is not allowed in strict mode. What should I do to correct it?

**Reply by Chengqian-Zhang** [reply]
I doubt that the deepmd-kit version and `input.json` version are not consistent. You can check your  deepmd-kit version and try using the corresponding `examples/water/dpa2/input_torch.json`.
